packages/shared/src/medical/fhir/observation-ordering.ts

```diff
@@ -0,0 +1,50 @@
+import { Observation } from "@medplum/fhirtypes";
+
+// Not sure the names of the codes are correct,
+// but they're related and should be grouped together regardless
+const OXYGEN_SATURATION_LOINC = {
```

Comment:
I suggest we name these constants using camelCase, following our [naming pattern](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#16cd72caa4944a10abfac5bf2d0a13e4).

Reply:
addressed